### PR TITLE
hover effects in your navbar change by awais alam

### PR DESCRIPTION
**Pull request by awaisalam909**
check then merge

### CODE REVIEW

File: _sass/jekyll-theme-modernist.scss [hunk 1]
```diff
@@ -149,11 +149,19 @@ header {
...
+    
       strong {
         font-size:14px;
         display:block;
         color:#fff;
         -webkit-font-smoothing:antialiased;
+        
+        ul li a:hover{
+          background:lime;
+          color:red;
```
Comment (bilalahmad99) [inline _sass/jekyll-theme-modernist.scss:162]: can we make colour a bit more suitable with the theme, e.g. grey, blue etc

**Review by bilalahmad99 — CHANGES_REQUESTED** [review]
change hover color to be suitable to theme

**Comment by awaisalam909** [comment]
bilal bhai wo docker wala idea add kr dy kindly

**Comment by awaisalam909** [comment]
ok I customz better colors

**Pull request closed without merging**